main.go

```diff
@@ -289,16 +344,16 @@ func generateTemplateGoCode(bt *BoardTemplate) (string, error) {
 		tpl += "\t\t\tHideMarkers: " + fmt.Sprint(qt.GraphSettings.HideMarkers) + ",\n"
 		tpl += "\t\t\tPreferOverlaidCharts: " + fmt.Sprint(qt.GraphSettings.OverlaidCharts) + ",\n"
 		tpl += "\t\t},\n"
-		tpl += "\t\tAutoFilter: true,\n"
+		tpl += "\t\tAutoFilter: false,\n"
```

Comment:
Since we are downloading exact board queries, we shouldn't need to manipulate filters